src/lpmd_proc.c

```diff
@@ -513,9 +526,11 @@ static int proc_message(message_capsul_t *msg)
 		case LPM_FORCE_OFF:
 			// Never enter LPM mode
 			exit_lpm (USER_EXIT);
+			poll_for_wlt(0);
 			break;
 		case LPM_AUTO:
 			// Enable oppotunistic LPM
+			poll_for_wlt(1);
```

Comment:
Maybe we should hook poll_for_wlt() call into enter_lpm/exit_lpm to cover more cases. Say, we should disable wlt polling when switch from LPM_AUTO to LPM_FORCE_ON.